openapi.yaml

```diff
@@ -5262,11 +5262,19 @@ components:
             (link to older version) and `successor-version` (link to newer version) can also be used
             to show the relation between versions.
 
+            2. `version-history`: A link to a changelog and/or a list of versions of the process
```

Comment:
will this duplicate  `2.` in the markdown give duplicated rendering of the "2" bullet?

Reply:
Good point, fixed.